src/en/dopebox/src/eu/kanade/tachiyomi/animeextension/en/dopebox/DopeBox.kt

```diff
@@ -86,6 +86,7 @@ class DopeBox : ConfigurableAnimeSource, ParsedAnimeHttpSource() {
             seasonsElements.forEach {
                 val seasonEpList = parseEpisodesFromSeries(it)
                 episodeList.addAll(seasonEpList)
+                episodeList = episodeList.reversed().toMutableList()
             }
```

Comment:
why reverse the entire episode list after every season element?
shouldnt we just reverse the list at the very end?

Reply:
Aah shit, my bad.

Sorry for getting back so late, was asleep.

Just made the updated the files with the fix, should be fine now, I hope.